samples/nrf9160/modem_shell/README.rst

```diff
@@ -538,6 +538,26 @@ Examples
 
 ----
 
+Running stored commands after startup
+=====================================
+
+MoSh command: ``startup_cmd``
+
+You can store up to three MoSh commands to run on start/bootup.
+By default, commands are run after the default PDN context is activated,
```

Comment:
Is this meant to be on two lines?

Reply:
I think it is ok now, not meant to be on 2 lines.